platform/keysystem/worldconfig/keyclient.go

```diff
@@ -91,28 +91,28 @@ func (b *ActionBuilder) DefaultDownloads(variant string) {
 		OneDay,
 		0644,
 	)
+	b.PublicKey(
+		"serviceaccount",
+		"/etc/homeworld/keys/serviceaccount.pem",
+		OneDay,
+	)
+	b.PublicKey(
+		"etcd-client",
+		"/etc/homeworld/authorities/etcd-client.pem",
+		OneDay,
+	)
+	b.PublicKey(
+		"etcd-server",
+		"/etc/homeworld/authorities/etcd-server.pem",
+		OneDay,
+	)
```

Comment:
Does downloading these public keys implicitly trust them?  If so then this change is making these keys trusted in more places than they were before.  (Of course there's really not much difference between "master-only" and "all nodes" but I still think it's worth pointing out)

Reply:
Only the keys in special directories like `/usr/local/share/ca-certificates/extra/`. Others are only pulled in by the particular services that need to validate them, which won't be running on the non-master nodes.